install_config/storage_examples/ceph_example.adoc

```diff
@@ -1,5 +1,5 @@
 [[install-config-storage-examples-ceph-example]]
-= Complete Example Using Ceph RBD
+= Using Ceph RBD for Persistent Storage
```

Comment:
Why capitalize "Persistent Storage?"
Please update the title in the topics.yml file.